components/gh_events/GitHubEvent.tsx

```diff
@@ -68,22 +81,32 @@ export default function GitHubEvent({ event }: { event?: IGitHubEvent }) {
       if (event.payload.review.state === "changes_requested")
         action = "requested changes on";
       title = (
-        <>
+        <div className="flex gap-1 items-center">
+          {action === "approved" && (
+            <GoCheck className="text-green-500 text-lg font-bold" />
+          )}
+          {action === "commented on PR" && (
+            <GoComment className="text-green-500 text-lg font-bold" />
+          )}
+          {action === "requested changes on" && (
+            <GoFileDiff className="text-green-500 text-lg font-bold" />
+          )}
           {action}{" "}
           <a
             className="cursor-pointer text-gray-300 font-bold"
             href={event.payload.review.html_url}
           >
             {event.repo.name}#{event.payload.pull_request.number}
           </a>
-        </>
+        </div>
       );
       body = <OpenGraphImage url={event.payload.pull_request.html_url} />;
       break;
 
     case "PushEvent":
       title = (
-        <>
+        <div className="flex gap-1 items-center">
+          <GoRepoPush className="text-green-500 text-lg font-bold" />
```

Comment:
Let's keep the colors of the icon the same or in line with the theme. We are not using this green anywhere else IIRC.

I'd prefer to go with the same color as the title text.